x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_bulk_body.ts

```diff
@@ -58,7 +58,10 @@ export const buildBulkBody = ({
     tags,
     throttle,
   });
-  const signal = buildSignal(doc, rule);
+  const signal: Signal = {
+    ...buildSignal([doc], rule),
```

Comment:
`buildSignal` now expects an array of documents and handles building the rule, parents, and ancestors. `additionalSignalFields` takes a single document and creates the fields for the special case where we only have 1 event triggering the signal (so we can copy the fields from the event into the signal). When we add signals based on EQL sequences we won't be able to copy every event from the sequence into the signal.